### PR TITLE
Add consistent_read parameter to Model.scan

### PR DESCRIPTION
**Pull request by lita**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 install_requires = [
     'six',
-    'botocore>=1.0.0',
+    'botocore>=1.2.0',
```
Comment (danielhochman) [inline setup.py]: can you update requirements.txt as well?

File: pynamodb/models.py [hunk 2]
```diff
@@ -723,6 +724,7 @@ def rate_limited_scan(cls,
             allow_rate_limited_scan_without_consumed_capacity=allow_rate_limited_scan_without_consumed_capacity,
             max_sleep_between_retry=max_sleep_between_retry,
             max_consecutive_exceptions=max_consecutive_exceptions,
+            consistent_read=consistent_read
```
Comment (danielhochman) [inline pynamodb/models.py]: can you use trailing commas?

File: setup.py (hunk 1 above)
Comment (lita) [inline setup.py]: yes!

File: pynamodb/models.py (hunk 2 above)
Comment (lita) [inline pynamodb/models.py]: yup

File: pynamodb/models.py [hunk 3]
```diff
@@ -765,7 +768,8 @@ def scan(cls,
             limit=page_size,
             scan_filter=key_filter,
             total_segments=total_segments,
-            conditional_operator=conditional_operator
+            conditional_operator=conditional_operator,
+            consistent_read=consistent_read
```
Comment (garrettheel) [inline pynamodb/models.py]: Can you document this new param in the methods too?

File: requirements.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
-botocore==1.1.2
+botocore>=1.2.0
```
Comment (danielhochman) [inline requirements.txt]: i would still pin to a specific version here

File: pynamodb/models.py (hunk 3 above)
Comment (lita) [inline pynamodb/models.py]: done!

**Pull request merged**